_team.qmd

```diff
@@ -0,0 +1,14 @@
+## Research Computing support team {.smaller}
```

Comment:
```suggestion
## Research Computing Support Team {.smaller}
```

I don't know if you could drop Support and just have it be `Research Computing Team`?

Reply:
```suggestion
## Research Computing Team {.smaller}
```